src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -40,8 +40,8 @@
 from google.protobuf import descriptor_pb2
 
 from src.proto.grpc.testing import empty_pb2
-
-from src.proto.grpc.testing.proto2 import empty2_pb2
+# empty2_pb2 is imported for import-consequent side-effects.
+from src.proto.grpc.testing.proto2 import empty2_pb2 # pylint: disable=unused-import
```

Comment:
I copied and pasted directly from your https://github.com/grpc/grpc/pull/10216#discussion_r107531453. done 👍